src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: CoreProtectTNT
 main: com.mcsunnyside.coreprotecttnt.Main
 version: 1.0
-api-version: 1.13
+api-version: 1.16
```

Comment:
Does the plugin use 1.16-specific APIs?

Reply:
i believe if it loads in non-repsawn_anchor versions will cause 
```
clickedBlock.getBlockData() instanceof RespawnAnchor
```
throw exception.

Reply:
Then try and catch the exception